gap/greens/gree.gi

```diff
@@ -391,8 +391,6 @@ function(S)
 
   l  := LeftCayleyGraphSemigroup(S);
   r  := RightCayleyGraphSemigroup(S);
-  # WW: in the future, when l and r are digraphs, gr can be created
-  #     by using DigraphEdgeUnion(l, r)
```

Comment:
Yes, because there are no methods for `Left/RightCayleyDigraph` for semigroups not in the category `IsEnumerableSemigroupRep` and so we use `Left/RightCayleyGraphSemigroup` which are defined in the library and hence not digraphs. So, basically the comment does not make sense in this method, although it does in the method for enumerable semigroups. Hence I deleted it.